pkg/resources/cruisecontrol/configmap.go

```diff
@@ -165,6 +184,20 @@ func GenerateCapacityConfig(kafkaCluster *v1beta1.KafkaCluster, log logr.Logger,
 	}
 	sort.Strings(brokerIdFromStatus)
 
+RemovePresentBrokerIds:
+	for _, presentBrokerId := range presentBrokerIds {
```

Comment:
I think `util.StringSliceRemove` could be used or an enhanced version of it.

Reply:
Just checked, it does literally the same thing but does look a lot neater, thanks :)